libevmasm/Assembly.h

```diff
@@ -170,6 +192,14 @@ class Assembly
 
 	unsigned codeSize(unsigned subTagSize) const;
 
+	/// Add all assembly items from given JSON array.
+	void addAssemblyItemsFromJSON(Json::Value const& _code);
```

Comment:
The docstring should mention that this method only works with a clean assembly that has no items. I would also rename it to remove that expectation:
```suggestion
	void importAssemblyItemsFromJSON(Json::Value const& _code);
```